flac.go

```diff
@@ -111,6 +130,33 @@ func (stream *Stream) parseStreamInfo() (isLast bool, err error) {
 	return block.IsLast, nil
 }
 
+// skipId3v2 skips ID3v2 data prepended to flac files.
+func (stream *Stream) skipId3v2() (err error) {
+	r := bufio.NewReader(stream.r)
+
+	// Discard unnecessary data from the ID3v2 header.
+	_, err = r.Discard(2)
+	if err != nil {
```

Comment:
Personal preference, keep scope of variable to a minimum if possible. This makes it clear we don't use the value later.
```go
if _, err := r.Discard(2) {
	return err
}
```